src/components/pages/WordCloud/CropCloud.js

```diff
@@ -38,7 +38,6 @@ export default function CropCloud() {
           src={`data:image/png;base64, ${placeholderCloud.data}`}
         />
       </div>
- main
```

Comment:
Could you explain the purpose of removing this line?

Reply:
"main" was appearing right in the middle of the page after the crop-cloud-container.